DOC Clarify Sphinx build instructions (#29743)
scikit-learn/scikit-learn: doc/developers/contributing.rst

    make html

This will run all the examples, which takes a while. If you only want to generate 

> images for a few examples, you can pass their names (or parts of) to the build. Such,
> the line below will run all example files starting from `plot_calibration`:

Reviewer: Just a small language tweak

```suggestion
images for a few examples, you can pass their names (or parts of) to the build command.
The line below will run all examples with names starting with `plot_calibration`:
```

> This will run all the examples, which takes a while. If you only want to generate 
> images for a few examples, you can pass their names (or parts of) to the build command.
> The line below will run all examples with names starting with `plot_calibration`:

Reviewer 2: ```suggestion
This will run all the examples, which takes a while. You can also run only a few examples based on their file names.
Here is a way to run all examples with file names containing with `plot_calibration`:
```

.. prompt:: bash

    EXAMPLES_PATTERN="plot_calibration" make html

> You can also pass regular expressions to gain more advanced control over examples

Reviewer 2: You can actually use regular expressions for more advanced use cases.

> You can also pass regular expressions to gain more advanced control over examples
> you want run.

Reviewer 2: ```suggestion
You can actually use regular expressions to have more control over which examples run.
```
